src/App.tsx

```diff
@@ -6,46 +6,76 @@ import { usePersistentState } from './PersistentState';
 import { useState, useEffect } from 'react';
 import './App.css';
 
+// truncate number to n decimal places
+const trunc = (numberToTruncate: number, numberOfDecimalPlaces: number) => {
```

Comment:
How about the `number.toFixed()` function? It would remove the need for a separate function for rounding. It would just always round a number to that decimal place if you care about that, i.e. `0` -> `0.00`. Examples below.

Reply:
yep, if I had spent 2 seconds to google if this already existed in javascript I would have found this and used it